fabrics.c

```diff
@@ -115,6 +115,16 @@ static void space_strip_len(int max, char *str)
 	}
 }
 
+static void set_discovery_kato(struct nvme_fabrics_config *cfg)
+{
+	/* Set kato to NVMF_DEF_DISC_TMO for persistent controllers */
+	if (persistent && !cfg->keep_alive_tmo)
```

Comment:
Where do you get the 'persistent' value from? Shouldn't it be rather an argument to the function?

Reply:
The persistent variable is a global bool. It's a bit ugly I agree. struct nvme_fabrics_config doesn't have it. We could add it there (libnvme) and also move this set_discovery_kato could move then there. Though I don't know if this makes sense.

Reply:
I've added the patch in the meantime as it fixes a regression. We can still cleanup ontop of this change